### ISSUE TITLE
NS records for parent domain's nameservers...subdomain is appended to FQDN

### ISSUE THREAD

**Issue by jdelaporte** [bertvv/ansible-role-bind#12]
So, my domain is a a child domain of my college's domain. I need to have NS records for the NS servers that are authoritatively serving records I transfer to them. I also transfer records to our ISP, who serves my zone authoritatively. I have the need to put their records in the file appearing like so:

                    IN  NS     ns1.mydomain.college.edu.
                     IN  NS     ns2.mydomain.college.edu.
                     IN  NS     dns1.college.edu..mydomain.college.edu.
                     IN  NS     dns2.college.edu..mydomain.college.edu.
                     IN  NS     ns1.ISP.net..mydomain.college.edu.
                     IN  NS     ns2.ISP.net..mydomain.college.edu.

I have thought about this for a bit, and it seems easiest to change the variable to be the FQDN, and not append the domain name at all in the template. 

As a reference, Zytrax.com states, "There is no requirement that any name servers are within the domain for which they are authoritative." http://www.zytrax.com/books/dns/ch8/ns.html

**Comment by jdelaporte** [comment]
The same issue occurs in the reverse zone files.

**Comment by bertvv** [comment]
I just pushed a proposal to the github repo. I introduced a variable `bind_other_name_servers` for name servers outside of your own domain. This:

```Yaml
    bind_zone_name_servers:
      - testbindmaster
      - testbindslave
    bind_other_name_servers:
      - ns1.isp.com
      - ns2.isp.com
```

then becomes (both in forward and reverse zone file):

```Yaml
                     IN  NS     testbindmaster.example.com.
                     IN  NS     testbindslave.example.com.
                     IN  NS     ns1.isp.com.
                     IN  NS     ns2.isp.com.
```

The template adds the dot so you don't have to.

**Comment by jdelaporte** [comment]
This worked perfectly for me. Thanks!

**Comment by bertvv** [comment]
This is now included in version 3.3.0 and newer.
